Sources/core/objects/Human.cpp

```diff
@@ -166,7 +170,14 @@ void Human::processGUImsg(const Message2 &msg)
                 }
                 
             }
-        } 
+            else if(id_ptr_on<Gun> tool = interface_.GetActiveHand().Get())
+            {
+		if(GetLying() == false && tool->Targetable(item))
+                {
```

Comment:
Static methods should be called via class name: `Gun::Targetable(item)`.
Tabs, spaces after `if`s.